File: src/main/java/io/github/gravetii/theme/ThemeType.java
package io.github.gravetii.theme;

public enum ThemeType {
  RANDOM,
  AZURE,
  BLACK_AND_WHITE,
  BLUEZY,
  EUANTHE,
  CONTRA,
  MOTLEY,
  MOSSY,
  SUNSET,
  TERRA,
  VERDURA,
  PATCHY,
  ;

  public String getShowableName() {
    return this.name().replace("_", " ");
  }

  public String getImgPath() {
    return "/theme/" + this.name().toLowerCase() + ".jpg";
  }

  public String getCssPath() {
    return "/theme/" + this.name().toLowerCase() + ".css";
  }
}

File: src/main/kotlin/io/github/gravetii/theme/ThemeType.kt
package io.github.gravetii.theme

enum class ThemeType {

    RANDOM,
    AZURE,
    BLACK_AND_WHITE,
    BLUEZY,
    EUANTHE,
    CONTRA,
    MOTLEY,
    MOSSY,
    SUNSET,
    TERRA,
    VERDURA,
    PATCHY,
    ;

    fun getDisplayName(): String = name.replace("_", " ")

    fun getImgPath(): String = "/theme/${name.lowercase()}.jpg"

    fun getStyleSheetPath(): String = "/theme/${name.lowercase()}.css"

}